scripts/parse.py

```diff
@@ -0,0 +1,112 @@
+##############################################################################
+#   AUTHOR: Krish Agarwal
+#   AFFILIATION: University_of_Basel
+#   AFFILIATION: Swiss_Institute_of_Bioinformatics
```

Comment:
As you are not officially an SIB member I would remove this line but keep only University of Basel, as you are working in our research group now.